entity-types/ext-keda/tests/Metric.stg.json

```diff
@@ -0,0 +1,53 @@
+[
+  {
+    "clusterName": "stg-harsh-mine",
+    "instrumentation.name": "nri-prometheus",
+    "instrumentation.provider": "newRelic",
+    "k8s.cluster.name": "stg-harsh-mine",
+    "label.app": "keda-operator-metrics-apiserver",
+    "label.app.kubernetes.io/component": "operator",
+    "label.app.kubernetes.io/instance": "keda-stg-stg-harsh-mine",
+    "label.app.kubernetes.io/name": "keda-operator-metrics-apiserver",
+    "metricName": "keda_build_info",
+    "namespaceName": "keda"
+  },
```

Comment:
This datapoint does not include `app_kubernetes_io_name` needed as the identifier to synthesise the entity